packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -1461,6 +1461,76 @@ const Conversation = WebexPlugin.extend({
     });
   },
 
+  /**
+   * Get all parent ids for a conversation.
+   * @param {string} conversationUrl conversation URL.
+   * @param {object} [query] object containing query string values to be appended to the url
+   * @param {string} [query.sinceDate] The date to get the parents from.
+   * @param {string} [query.maxDate] The parents published date is before this date
+   * @param {Number} [query.includeParentCount] max number of parents to return.
+   * @param {string} [query.activityType] The type of children to return the parents of, a null value here returns parents of all types of children.
+   * The value is one of 'reply', 'edit', 'cardAction', 'reaction', 'reactionSummary', 'reactionSelfSummary'
+   * @returns {Promise<Array<String>>}
+   */
+  async listParentActivityIds(conversationUrl, query) {
+    const params = {
+      method: 'GET',
+      url: `${conversationUrl}/parents`,
+      qs: query
+    };
+
+    const response = await this.request(params);
+
+    return response.body;
+  },
+
+  /**
+   * Return a list of child activities with a given conversation, parentId and other constraints.
+   * @param {string} conversationUrl targeted conversation URL
+   * @param {string} activityParentId parent id of edit activities or thread activities
+   * @param {string} activityType type of child activity to return. One of 'reply', 'edit', 'reaction'
+   * The value is one of 'reply', 'edit', 'cardAction', 'reaction', 'reactionSummary', 'reactionSelfSummary'
+   * @param {?object} query object containing query string values to be appended to the url
+   * @param {number} [query.limit] The limit of child activities that can be returned per request
+   * @param {boolean} [query.latestActivityFirst] Sort order for the child activities
+   * @param {boolean} [query.includeParentActivity] Enables the parent activity to be returned in the activity list
+   * @param {string} [sinceDate] Get all child activities after this date
+   * @param {string} [maxDate] Get all child activities before this date
+   * @returns {Promise<Array>}
+   */
+  async listChildActivitiesByParentId(conversationUrl, activityParentId, activityType, query = {}) {
+    const finalQuery = {
+      ...query,
+      activityType
+    };
+    const params = {
+      method: 'GET',
+      url: `${conversationUrl}/parents/${activityParentId}`,
+      qs: finalQuery
+    };
+
+    return this.request(params);
+  },
+
+  /**
+   * Return an array of reactionSummary and reactionSelfSummary objects
+   * @param {string} conversationUrl targeted conversation URL
+   * @param {string} activityParentId parent id of reaction activities
+   * @param {object} query object representing query parameters to pass to convo endpoint
```

Comment:
Is this the same query object as `listChildActivitiesByParentId`? If so, can we `@typedef` the query object?